.circleci/config.yml

```diff
@@ -31,7 +31,8 @@ jobs:
           paths:
             - node_modules
       - run: yarn run gen-snapshot-tests
-      - run: yarn run build:all
+      - run: yarn run build:cjs
+      - run: yarn run build:es6
```

Comment:
Why do we have to build these in circleci? (Honest question, It's not clear to me if they're being used at all.)